spec/models/audit/event/authn/inject_client_cert_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+
+describe Audit::Event::Authn::InjectClientCert do
+  let(:role_id) { 'rspec:host:my_host' }
+  let(:role) { double('the-role', id: role_id) }
+  let(:authenticator_name) { 'the-authenticator'}
+  let(:service) { double('the-service', resource_id: 'rspec:webservice:the-service') }
+  let(:success) { true }
+  let(:error_message) { nil }
+
+  subject do
+    Audit::Event::Authn::InjectClientCert.new(
+      role: role,
+      authenticator_name: authenticator_name,
+      service: service,
+      success: success,
+      error_message: error_message
+    )
+  end
+
+  context 'when successful' do
+    it 'produces the expected message' do
+      expect(subject.message)
+        .to eq('rspec:host:my_host successfully injected client certificate with' + \
+               ' authenticator the-authenticator service rspec:webservice:the-service')
+    end
+
+    it 'uses the INFO log level' do
+      expect(subject.severity).to eq(Syslog::LOG_INFO)
+    end
+
+    it 'renders to string correctly' do
+      expect(subject.to_s).to eq('rspec:host:my_host successfully injected client' + \
+                                 ' certificate with authenticator the-authenticator' + \
+                                 ' service rspec:webservice:the-service')
```

Comment:
A pure formatting nitpick, but one I have medium-strong opinion about.  I prefer:
```ruby
expect(subject.to_s).to eq(
  'rspec:host:my_host successfully injected client certificate with '
  'authenticator the-authenticator service rspec:webservice:the-service'
)
```
This is a general preference vs keeping the indentation level of the opening paren.  Here are my reasons:
1. The indentation level of the open paren is arbitrary, based on however long the line was up until you open the paren that will take the long string (or anything else).
2. At a page level, when there are multiple instances, this creates a ragged look.
3. In the extreme case, what if you open your paren at 75 chars?  then you have a bunch of 5 character lines pushed over to the right?
4. Versus restarting the line as in my example, which allows you to take up the full line width, minus only a 2 space indent to show the nesting.